specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -2465,7 +2465,10 @@
             "Creating",
             "Active",
             "Deleting",
-            "Disabled"
+            "Disabled",
+            "Migrating",
+            "Migrated",
+            "PendingMigrationCommit"
```

Comment:
If we are doing rollback, We need a state to say that the profile is rolling back